src/ffi/orx.ml

```diff
@@ -48,8 +48,8 @@ let create_exn create_from_config what name =
 type camera = Orx_gen.Camera.t
 type obj = Orx_gen.Object.t
 
-module Color = Orx_gen.Color
-module Display = Orx_gen.Display
+module _ = Orx_gen.Color
```

Comment:
These `module _ = ...` modules are not usefully exposed in the module interface. I don't want to delete the bindings there yet as they can be built on, but for now there is not much purpose in exposing what exists publicly.